build/filters.js

```diff
@@ -89,7 +89,9 @@ module.exports.indentationFilter = [
 	'!test/monaco/out/**',
 	'!test/smoke/out/**',
 	'!extensions/typescript-language-features/test-workspace/**',
+	'!extensions/typescript-language-features/resources/walkthroughs/**',
```

Comment:
Ordering alphabetically and cleaning this up a bit - I did this when I thought it was a hygiene error and it's helpful to keep things in this file organized.